_IBC/neutron-carbon.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
```

Comment:
the chains but be numbered in alphabetical order. so the file name and the chain_1 and chain_1 should be swapped so that carbon comes first.